macaw-riscv/src/Data/Macaw/RISCV.hs

```diff
@@ -0,0 +1,69 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Data.Macaw.RISCV (
+  module Data.Macaw.RISCV.Disassemble,
+  module Data.Macaw.RISCV.Eval,
+  module Data.Macaw.RISCV.Identify,
+  module Data.Macaw.RISCV.RISCVReg,
```

Comment:
Here I exposed the entire modules that are no longer exposed in the Cabal file. However, this might be exposing too much of the internals? What is reasonable to expose and hide here?

Reply:
We should be able to elide Disassemble, Eval, and Identify safely. RISCVReg probably needs to be exposed